server/app.js

```diff
@@ -6,16 +6,21 @@ const model = require('../database/models/Room.js');
 const app = express();
 
 app.use(morgan('dev'));
-
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.use(express.static(path.join(__dirname, '../client/dist')));
-
 app.get('/rooms/:id', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/dist/index.html'));
```

Comment:
Now I see what you meant about the extra GET needed to serve up the html but it's still unclear to me whether the typical way we've served the initial html (e.g. adding 'app.use(express.static('public'));' in the server) is equivalent. Is there an issue i'm not seeing that would make it bad in this particular case or just two ways of achieving the same thing?

Reply:
@juuprz yeah I thought your way (`using express.static()`) would work but for some reason I couldn't get this to work for me so had to set up individual get handlers. will file a tech debt ticket to try and tackle this later.